src/Web/Scotty/Action.hs

```diff
@@ -29,72 +29,65 @@ module Web.Scotty.Action
     , stream
     , text
     , Param
-    , Parsable(..)
-      -- private to Scotty
+    , Parsable (..)
+    -- private to Scotty
     , runAction
     ) where
 
-import           Blaze.ByteString.Builder   (fromLazyByteString)
-
-import qualified Control.Exception          as E
-import           Control.Monad              (liftM, when)
-import           Control.Monad.Error.Class
-import           Control.Monad.IO.Class     (MonadIO(..))
-import           Control.Monad.Reader       (MonadReader(..), ReaderT(..))
+import Control.Monad.Error.Class
+import Control.Monad.Trans.Except
+import Data.Int
+import Data.Word
+import Network.HTTP.Types
+import Network.Wai
+import Numeric.Natural
+import Prelude.Compat
+
+import Blaze.ByteString.Builder (fromLazyByteString)
+import Control.Monad (when)
+import Control.Monad.Reader (MonadIO (..), MonadReader (..), ReaderT (..), asks)
+import Data.Default.Class (def)
+import Data.Text.Lazy.Encoding (encodeUtf8)
+import Prelude ()
+
+import qualified Control.Exception as E
 import qualified Control.Monad.State.Strict as MS
-import           Control.Monad.Trans.Except
-
-import qualified Data.Aeson                 as A
-import qualified Data.ByteString.Char8      as B
+import qualified Data.Aeson as A
+import qualified Data.ByteString.Char8 as B
 import qualified Data.ByteString.Lazy.Char8 as BL
-import qualified Data.CaseInsensitive       as CI
-import           Data.Default.Class         (def)
-import           Data.Int
-import qualified Data.Text                  as ST
-import qualified Data.Text.Encoding         as STE
-import qualified Data.Text.Lazy             as T
-import           Data.Text.Lazy.Encoding    (encodeUtf8)
-import           Data.Word
-
-import           Network.HTTP.Types
--- not re-exported until version 0.11
-#if !MIN_VERSION_http_types(0,11,0)
-import           Network.HTTP.Types.Status
-#endif
-import           Network.Wai
-
-import           Numeric.Natural
-
-import           Prelude ()
-import           Prelude.Compat
-
-import           Web.Scotty.Internal.Types
-import           Web.Scotty.Util
+import qualified Data.CaseInsensitive as CI
+import qualified Data.Text as ST
+import qualified Data.Text.Encoding as STE
+import qualified Data.Text.Lazy as T
+
+import Web.Scotty.Internal.Types
+import Web.Scotty.Util
 
 -- Nothing indicates route failed (due to Next) and pattern matching should continue.
 -- Just indicates a successful response.
 runAction :: (ScottyError e, Monad m) => ErrorHandler e m -> ActionEnv -> ActionT e m () -> m (Maybe Response)
 runAction h env action = do
-    (e,r) <- flip MS.runStateT def
-           $ flip runReaderT env
-           $ runExceptT
-           $ runAM
-           $ action `catchError` (defH h)
+    (e, r) <-
+        flip MS.runStateT def $
+            flip runReaderT env $
+                runExceptT $
+                    runAM $
+                        action `catchError` defH h
```

Comment:
I don't necessarily agree with the code formatter, but I think it's useful for reducing future diff noise. I'm open to rolling back the formatting changes if people disagree.